src/main/resources/application.yaml

```diff
@@ -0,0 +1,10 @@
+server:
+  servlet:
+    context-path: /chd-order-api
```

Comment:
Any particular reason why we are setting this?

Reply:
It's me being on autopilot when going through the setup...